WTWSMS/history/characters/kurdish.txt

```diff
@@ -112,6 +112,7 @@
 	dynasty=1029098
 	religion="zoroastrian"
 	culture="kurdish"
+	trait=zurvanist_trait
```

Comment:
Is this historical or random?

Reply:
> 
> 
> Is this historical or random?

This was remnant of the school traits, which i added previously.